core/src/main/scala/sbt/librarymanagement/CrossVersionExtra.scala

```diff
@@ -5,19 +5,23 @@ import sbt.librarymanagement.syntax._
 
 final case class ScalaVersion(full: String, binary: String)
 
-private[librarymanagement] abstract class CrossVersionFunctions {
+private[librarymanagement] trait CrossVersionCompatAliases { self: CrossVersionFunctions =>
 
   /** Compatibility with 0.13 */
-  final val Disabled = sbt.librarymanagement.Disabled
-  final val Binary = sbt.librarymanagement.Binary
-  final val Constant = sbt.librarymanagement.Constant
-  final val Full = sbt.librarymanagement.Full
-  final val Patch = sbt.librarymanagement.Patch
+  final val Disabled = sbt.librarymanagement.Disabled()
```

Comment:
The corresponding construct in sbt 0.13 https://github.com/sbt/sbt-zero-thirteen/blob/v0.13.17/ivy/src/main/scala/sbt/CrossVersion.scala#L19:

```scala
  object Disabled extends CrossVersion { override def toString = "disabled" }
```

so the above change could be considered valid for source compat with sbt 0.13, but in general we probably should recommend migrating to lowercase `CrossVersion.disabled`.